mne/viz/_brain/_timeviewer.py

```diff
@@ -841,6 +843,14 @@ def configure_menu(self):
         menu = self.main_menu.addMenu('Help')
         menu.addAction('Show MNE key bindings\t?', self.help)
 
+    def configure_status_bar(self):
+        from PyQt5.QtWidgets import QLabel, QProgressBar
+        self.status_msg = QLabel(self.default_status_bar_msg)
+        self.status_progress = QProgressBar()
```

Comment:
The progress bar is currently unused, right? Are you planning on reviving it (+1 on that btw!), or why is it here? :)

Reply:
The progress bar is currently unused so it is hidden. But I add it in here because I plan to use it when I will add support for `save_movie` in `_TimeViewer`. It's an item of https://github.com/mne-tools/mne-python/issues/7162

Reply:
Thanks for the clarification. Just wanted to ensure you didn't accidentally leave sth in that you don't need anymore.